jekyll/_cci2/install-machine-runner-3-on-linux.adoc

```diff
@@ -3,17 +3,14 @@ contentTags:
   platform:
   - Cloud
 ---
-= Install machine runner 3.0 on Linux - Open preview
+= Install machine runner 3.0 on Linux
 :page-layout: classic-docs
 :page-liquid:
 :page-description: Instructions on how to install CircleCI's self-hosted machine runner (3.0) on Linux.
 :icons: font
-:experimental:
```

Comment:
Experimental tag is for docs engine features, not in relation to the doc itself being experimental.